networkx/tests/test_relabel.py

```diff
@@ -184,8 +184,8 @@ def test_relabel_selfloop(self):
         G = nx.relabel_nodes(G, {1: 0}, copy=False)
         assert_nodes_equal(G.nodes(), [0])
 
-#    def test_relabel_multidigraph_inout_inplace(self):
-#        pass
+    #    def test_relabel_multidigraph_inout_inplace(self):
```

Comment:
I don't know about this change either (though the commented test should probably just be removed)

Reply:
Removed.